spec/features/user_views_past_orders_spec.rb

```diff
@@ -0,0 +1,31 @@
+require "rails_helper"
+
+RSpec.describe "User views past orders" do
+
+  context "with valid authorization" do
+    let(:user) { User.create(full_name: "example",
+                                 email: "example@example.com",
+                              password: "password") }
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Expression at 8, 53 should be on its own line.